src/GuardClauses/GuardClauses.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
```

Comment:
Again, why?
By targeting 1.1, this package can be consumed by a much broader audience of apps. The higher the version of netstandard a library targets, the fewer apps that can consume it. Was there something missing from 1.1 that you needed for this PR?

Reply:
InvalidEnumArgumentException is not in version 1.1

Reply:
@ardalis 
That's what we discussed in the this issue https://github.com/ardalis/GuardClauses/issues/66#issuecomment-629707037

For .NET Framework I'm not sure it would be a problem to target netstandard2.0. .NET Framework project under v4.7.2 could still use the build targeting net40. 

For .NET Core, versions under 2.1 are not maintained any more ( https://dotnet.microsoft.com/platform/support/policy/dotnet-core ), but are probably still used in "old" projects.

For other implementation (Xamarin, UWP, ...), I don't know, I haven't any experience with these.
https://github.com/dotnet/standard/blob/master/docs/versions.md

Anyway, this is still a big breaking change. I'm not sure it worth it

Reply:
My thinking is that we're about to hit .NET 5 and the bulk of the GuardClauses functionality is pretty stable. If folks are on older versions of .NET, they can keep using the older versions that support that platform.

It would certainly help if I had some stats on what % of users of this package need netstandard1.1 support and can't consume 2.0. But I don't have that. I'm guessing most users are on Core and are keeping up with Microsoft's LTS versions, and 2.0 works fine in that scenario at this point.